art/attacks/evasion/projected_gradient_descent/__init__.py

```diff
@@ -0,0 +1,7 @@
+from art.attacks.evasion.projected_gradient_descent.projected_gradient_descent import ProjectedGradientDescent
+from art.attacks.evasion.projected_gradient_descent.projected_gradient_descent_numpy import \
+    ProjectedGradientDescentNumpy
+from art.attacks.evasion.projected_gradient_descent.projected_gradient_descent_pytorch import \
+    ProjectedGradientDescentPytorch
+from art.attacks.evasion.projected_gradient_descent.projected_gradient_descent_tensorflow_v2 import \
+    ProjectedGradientDescentTensorflowV2
```

Comment:
I think we should leave this file empty and use internally the full import paths instead.
```suggestion
```